wcfsetup/install/files/lib/data/user/avatar/Gravatar.class.php

```diff
@@ -98,13 +99,20 @@ public function getURL($size = null) {
 	 */
 	public static function test($email) {
 		$gravatarURL = sprintf(self::GRAVATAR_BASE, md5(mb_strtolower($email)), 80, GRAVATAR_DEFAULT_TYPE);
+		$client = HttpFactory::getDefaultClient();
+		$request = new Request('GET', $gravatarURL);
+		
 		try {
-			$tmpFile = FileUtil::downloadFileFromHttp($gravatarURL, 'gravatar');
-			@unlink($tmpFile);
-			return true;
+			$response = $client->send($request);
```

Comment:
```suggestion
			$response = $client->sendRequest($request);
```

`sendRequest` is the standardized PSR-7 method. Apply to the other locations as well, please.

Reply:
I will postpone this change to: https://github.com/WoltLab/WCF/issues/3623